src/NBitcoin/Network.cs

```diff
@@ -230,6 +230,102 @@ internal static Network Register(Network network)
             return network;
         }
 
+        /// <summary>
+        /// Mines a new genesis block, to use with a new network.
+        /// Typically, 3 such genesis blocks need to be created when bootstrapping a new coin: for Main, Test and Reg networks.
+        /// </summary>
+        /// <param name="consensusFactory">
+        /// The consensus factory used to create transactions and blocks. 
+        /// Use <see cref="PosConsensusFactory"/> for proof-of-stake based networks.
+        /// </param>
+        /// <param name="pszTimestamp">
+        /// Traditionally a news headline from the day of the launch, but could be any string or link.
+        /// This will be inserted in the input coinbase transaction script.
+        /// </param>
+        /// <param name="target">
+        /// The difficulty target under which the hash of the block need to be. 
+        /// Make the Test and Reg targets ones easier to find than the Main one so that you don't wait too long to mine the genesis block.
```

Comment:
Same here – it should be explained, which values are meaningful.

Reply:
I'm already providing a code sample and telling the user to make it easier for Test and Reg.
How much more help should I give to someone who wants to start a coin but doesn't understand how to make a target easier or harder 🤣

Reply:
I would appreciate little more info, but it is not required if you don't want it.

Reply:
added